.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "prettier.eslintIntegration": true,
+  "eslint.autoFixOnSave": true,
```

Comment:
According to my research, using `editor.formatOnSave: true` here will enable "format on save" feature using prettier (instead of removing prettier to use eslint auto fix). Can you give it a shot?

Reply:
I'll rollback the prettier changes

Reply:
@mondaychen 
Works like a charm!